src/1_create_canonical.jl

```diff
@@ -23,7 +23,8 @@ rme_ids = CSV.read(
     header=false,
     comment="#"
 )
-gbr_features = GDF.read(joinpath(DATA_DIR, "gbr_features", "Great_Barrier_Reef_Features.shp"))
+#Update to only inlude Reef Features --> "Reef" in FEAT_NAME, results in 3862 reef polygons
+gbr_features = filter(row -> row.FEAT_NAME == "Reef", GDF.read(joinpath(DATA_DIR, "gbr_features", "Great_Barrier_Reef_Features.shp")))
```

Comment:
Add spacing for readability

```suggestion

#Update to only inlude Reef Features --> "Reef" in FEAT_NAME, results in 3862 reef polygons
gbr_features = filter(row -> row.FEAT_NAME == "Reef", GDF.read(joinpath(DATA_DIR, "gbr_features", "Great_Barrier_Reef_Features.shp")))
```